index.js

```diff
@@ -144,23 +149,27 @@ class ServerlessDynamodbLocal {
     }
 
     startHandler() {
-        const config = this.config;
-        const options = _.merge({
-                sharedDb: this.options.sharedDb || true
-            },
-            config && config.start,
-            this.options
-        );
-
-        dynamodbLocal.start(options);
+        if (!this.options.noStart) {
+          const config = this.config;
+          const options = _.merge({
```

Comment:
The `options` variable is only declared inside the `if` scope. This results in an undefined `options` variable in line 164.